docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -0,0 +1,142 @@
+
+### Static Policy
+
+The `static` policy allows containers in Guaranteed pods with integer CPU
+requests to be assigned to exclusive CPUs on the node.  This exclusivity is
+enforced through the use of the
+[cpuset cgroup controller](https://www.kernel.org/doc/Documentation/cgroup-v1/cpusets.txt).
+
+**Note:** System services such as the container runtime and the kubelet itself can continue to run on these exclusive CPUs.  The exclusivity only extends to other pods.
+{: .note}
+
+The policy manages a shared pool of CPUs that initially contains all CPUs the
+node minus any reservations by the kubelet `--kube-reserved` or
+`--system-reserved` options.  CPUs reserved by these options are taken, in
+integer quantity, from the initial shared pool in ascending order by physical
+core ID.  This shared pool is the set of CPUs on which any containers in
+BestEffort and Burstable pods run. Containers in Guaranteed pods with fractional
+cpu requests also run on CPUs in the shared pool.  Only containers that are
+both part of a Guaranteed pod and have integer CPU requests are assigned
+exclusive CPUs.
+
+**Note:** When reserving CPU with `--kube-reserved` or `--system-reserved` options, it is advised to use *integer* CPU quantities.
+{: .note}
+
+As Guaranteed pods whose containers fit the requirements for being statically
+assigned are scheduled to the node, CPUs are removed from the shared pool and
+placed in the cpuset for the container.  CFS quota is not used to bound
+the CPU usage of these containers as their usage is bound by the scheduling domain
+itself i.e. the number of CPUs in the container cpuset is equal to the integer
+CPU limit specified in the pod spec.  This static assignment increases CPU
+affinity and decreases context switches due to throttling for the CPU bound
+workload.
+
+In the event that the shared pool is depleted the kubelet takes two actions:
+* Evict all pods that include a container that does not have a CPU request as
+  those pods now have no CPUs on which to run 
+* Set a `NodeCPUPressure` node condition to `true` in the node status. When
+  this condition is true, the scheduler will not assign any pod to the node
+  that has a container without a CPU request.
+
+Consider the containers in the following pod specs:
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```yaml
+
+**Result** BestEffort QoS because no resource requests or limits are specified.
+Will be evicted if shared pool is depleted. Runs in the shared pool.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+      requests:
+        memory: "100Mi"
+```yaml
+
+**Result** Burstable QoS because resource requests != limits and CPU is not
+specified.  Will be evicted if shared pool is depleted. Runs in the shared pool.
+
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+	cpu: "2"
```

Comment:
I'm curious to see how these red blocks render in an otherwise functional code block. Please @mention me after removing `yaml` and rebuilding the preview.